apiary.apib

```diff
@@ -1702,8 +1702,114 @@ root will not be listable, as it is virtual. Access known paths directly.
 # Group resolve
 
 # Group stats
+Query IPFS statistics.
 
-## bw
+This command can't be called directly.
+
+## bw [POST /stats/bw{&peer,proto,poll,interval}]
+Print ipfs bandwidth information.
+
++ Parameters
+    + peer (string, optional) - Specify a peer to print bandwidth for.
+    + proto (string, optional) - Specify a protocol to print bandwidth for.
```

Comment:
Not an issue that is related directly to this subcommand, so going to ignore it for now. An issue for global documentation has been opened here https://github.com/ipfs/go-ipfs/issues/2583.